Reorganize `astro workspace` docs by breaking out subcommands into separate docs (#1978)
astronomer/docs: astro/cli/astro-workspace-user-update.md

---
sidebar_label: "astro workspace user update"
title: "astro workspace user update"
id: astro-workspace-user-update
description: Reference documentation for astro workspace user update.
hide_table_of_contents: true
---

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

:::info  

The behavior and format of this command differs depending on what Astronomer product you're using. Use the following tabs to change product contexts. 

:::

<Tabs
    defaultValue="astro"
    values={[
        {label: 'Astro', value: 'astro'},
        {label: 'Software', value: 'software'},
    ]}>
<TabItem value="astro">

> Update a user's permissions in your current Astrp Workspace. The CLI prompts you for the user's email.

Reviewer: ```suggestion
Update the role of an existing user in your current Workspace. The CLI prompts you for the user's email.
```

Does it prompt you for other commands? Should we specify that in the other docs?

Author: So weirdly this command does not prompt you for the role, only for the email. If you select an email and you don't include a role, it throws an error

## Usage

```sh
astro workspace user update --role <user-role>
```

## Options

| Option                | Description                                          | Valid Values                                                                     |
| --------------------- | ---------------------------------------------------- | -------------------------------------------------------------------------------- |

> | `<email>`             | The email for the user you want to invite or update. | Any valid email                                                                  |

Reviewer: ```suggestion
| `<email>`             | The email address of the user whose role you want to update. | Any valid email                                                                  |
```
